### ISSUE TITLE
Text missing from 2.2.2 \DeclareBoolOption

### ISSUE THREAD

**Issue by hsmyers** [ho-tex/kvoptions#9]
The missing command `\ProcessKeyvalOptions*` (or the like) should appear before you say the user is ready with the if\else\fi statements.

**Comment by u-fischer** [comment]
The documentation is not about the handling and processing of LaTeX options. Check the LaTeX documentation for this. Beside this there is a full example in section 3.

**Comment by hsmyers** [comment]
The text is not mentioned until line 74 of the example. In my estimation,
it should be mentioned where it is needed. This would correct the existing
text that clearly states:
>
> *Now the switch is ready to use in the package/class, e.g.:*

Why is there disagreement over at most a line or two?

**Comment by u-fischer** [comment]
Well if you find it so important, create a pull request.

**Comment by hsmyers** [comment]
No. Given your response, why would I?

**Comment by u-fischer** [comment]
> No. Given your response, why would I?

Well I don't much time currently, and actually I already spent too much on finding out what your issue is about--you didn't put much effort in describing what you actually want. It would help if you wouldn't only require a change but also spent some of your time to make it real.

**Comment by hsmyers** [comment]
True, the request was phrased in what I thought was wrong, and not in
suggesting a solution. My apologies. I'll investigate your suggestion of a
pull request.

**Comment by hsmyers** [comment]
I made a local clone. The revision looks like this:

[image: kvoptions revision.png]
What now?
